core/lib/workarea/elasticsearch/query_cache.rb

```diff
@@ -102,6 +102,33 @@ def call(env)
         end
       end
 
+      # Execute sidekiq jobs with query cache enabled.
+      # Only enabled for workers that set use_elasticsearch_query_cache to true.
+      #
+      # @return [ Object ] The result of the call.
+      #
+      class SidekiqMiddleware
```

Comment:
I kinda feel like this should be in **app/middleware** but i might be in the minority :P

Reply:
The other middleware for query cache is in this file to, thought I'd be consistent.

Reply:
now that’s what I call consistent.